pkg/installation/install_test.go

```diff
@@ -151,6 +153,14 @@ func Test_removeLink_notExists(t *testing.T) {
 	}
 }
 
+func TestRemove_cantUninstallItself(t *testing.T) {
+	envPath := environment.MustGetKrewPaths()
+	expectedErrorMessagePart := "not allowed"
+	if err := Remove(envPath, "krew"); !strings.Contains(err.Error(), expectedErrorMessagePart) {
+		t.Fatalf("wrong error message for 'remove krew' action")
```

Comment:
Ah sorry for the back and forth. 

But you used Fatalf here without %s. 

Do you mind adding expected=%q; got=%q